Source/iOS/Extensions/UIDevice+Model.swift

```diff
@@ -33,4 +33,16 @@ public extension UIDevice {
 
     return result
   }
+    
+    public static func isIPhone() -> Bool {
```

Comment:
Maybe its just me but iPhone with a capital `I` kinda makes me sad. Don't get me wrong, I know why you wrote it that way.

Could we do `isPhone` and `isPad` instead?

Probably overreacting here :sunglasses: